js/main.js

```diff
@@ -0,0 +1,214 @@
+  }
+}
+
+function generateThumb(response) {
+  var recipeObject = response[0];
+  var page = data.currentDB.substring(0, data.currentDB.length - 1);
+  var bookmark = 'images/Empty_Bookmark.svg';
+  if (data['bookmarked' + data.currentDB].indexOf(recipeObject['id' + page]) >= 0) { bookmark = 'images/Checked_Bookmark.svg'; }
+  var heart = 'images/Empty_Heart.svg';
+  if (data['loved' + data.currentDB].indexOf(recipeObject['id' + page]) >= 0) { heart = 'images/Filled_Heart.svg'; }
+
+  var $recipeThumb = document.createElement('div');
+  $recipeThumb.className = 'swap recipe-thumb ' + page.toLowerCase() + '-border';
+  $recipeThumb.setAttribute('data-view', data.currentPage);
+  $recipeThumb.setAttribute('id', recipeObject['id' + page]);
+
+  var $leftImg = document.createElement('div');
+  $leftImg.className = 'left-img thumb-padding';
+  var $itemThumb = document.createElement('img');
+  $itemThumb.className = 'img-thumb';
+  $itemThumb.setAttribute('alt', 'picture of ' + page.toLowerCase());
+  $itemThumb.setAttribute('src', recipeObject['str' + page + 'Thumb']);
+  $leftImg.appendChild($itemThumb);
+
+  var $centerBrief = document.createElement('div');
+  $centerBrief.className = 'center-brief thumb-padding';
+  var $dishName = document.createElement('h2');
+  $dishName.textContent = recipeObject['str' + page];
+  var $subTitle1 = document.createElement('p');
+  var $describe1 = document.createElement('i');
+  var $subTitle2 = document.createElement('p');
+  var $describe2 = document.createElement('i');
+  if (data.currentDB === 'Meals') {
+    $subTitle1.textContent = 'Origin: ';
+    $describe1.textContent = recipeObject.strArea;
+    $subTitle2.textContent = 'Category: ';
+    $describe2.textContent = recipeObject.strCategory;
+  } else {
+    $subTitle1.textContent = 'Category: ';
+    $describe1.textContent = recipeObject.strCategory;
+    $subTitle2.textContent = 'Type: ';
+    $describe2.textContent = recipeObject.strAlcoholic;
+  }
+  $subTitle1.appendChild($describe1);
+  $subTitle2.appendChild($describe2);
+  $centerBrief.append($dishName, $subTitle1, $subTitle2);
+
+  var $rightbtns = document.createElement('div');
+  $rightbtns.className = 'right-bookmark';
+  var $bookmarkBtn = document.createElement('button');
+  $bookmarkBtn.className = 'bookmark-trigger seamless-btn';
+  var $bookmark = document.createElement('img');
+  $bookmark.setAttribute('src', bookmark);
+  $bookmark.setAttribute('alt', 'Bookmark');
+  $bookmarkBtn.append($bookmark);
+  var $lovedBtn = document.createElement('button');
+  $lovedBtn.className = 'loved-trigger seamless-btn';
+  var $loved = document.createElement('img');
+  $loved.className = 'full-center';
+  $loved.setAttribute('src', heart);
+  $loved.setAttribute('alt', 'Love');
+  $lovedBtn.append($loved);
+  $rightbtns.append($bookmarkBtn, $lovedBtn);
+
+  $recipeThumb.append($leftImg, $centerBrief, $rightbtns);
+  gallery.appendChild($recipeThumb);
```

Comment:
You should not append the DOM tree directly inside of the function like this. We want our functions to be reusable, and that makes this function only usable to append to this specific element. Instead, you should return the value from the function, and append it where the function is called.